src/commands/shared.ts

```diff
@@ -20,19 +20,16 @@ export const isPrivateKeyOption = (option: any): option is PrivateKeyOption => {
   return typeof option?.key === "string" || typeof option?.keyFile === "string";
 };
 
-export type AwsKmwSignerOption = {
+export type AwsKmsSignerOption = {
```

Comment:
Just to double check, this is a typo yea?

Reply:
yeah